### PR TITLE
model: impl `KethMaybeRelocatable` and `KethOption`

### PR DESCRIPTION
**Pull request by tcoratger**
Related https://github.com/kkrt-labs/keth/issues/37

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+scarb nightly-2024-09-28
```
Comment (tcoratger) [inline .tool-versions:1]:
```suggestion
```

File: crates/exex/src/model.rs [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    }
+}
+
+impl From<u64> for KethMaybeRelocatable {
+    fn from(value: u64) -> Self {
+        Self(Felt252::from(value).into())
+    }
+}
+
+/// [`KethOption`] is a custom representation of a Rust [`Option<u64>`] type, specifically tailored
```
Comment (ClementWalter) [inline crates/exex/src/model.rs:70]: can't we make this `Option<T: Into<KethMaybeRelocatable>>` ?

**Review by ClementWalter — APPROVED** [review]
left a comment to be more generic but ok as is

File: crates/exex/src/model.rs (hunk 2 above)
Comment (tcoratger) [reply crates/exex/src/model.rs]: Yes good idea, just fixed

**Comment by tcoratger** [comment]
> left a comment to be more generic but ok as is

Just fixed the comment, this is nicer

**Review by ClementWalter — APPROVED** [review]

**Pull request merged**